core/src/test/scala/org/apache/spark/deploy/RPackageUtilsSuite.scala

```diff
@@ -60,11 +64,9 @@ class RPackageUtilsSuite extends SparkFunSuite with BeforeAndAfterEach {
     }
   }
 
-  def beforeAll() {
-    System.setProperty("spark.testing", "true")
-  }
-
   override def beforeEach(): Unit = {
+    super.beforeEach()
+    System.setProperty("spark.testing", "true")
```

Comment:
what's the difference between having it in beforeEach and beforeAll for this specific config?